core/shared/src/main/scala/doodle/algebra/ClipIt.scala

```diff
@@ -0,0 +1,18 @@
+package doodle
+package algebra
+
+import doodle.core.ClosedPath
+import doodle.algebra.Picture
+
+trait ClipIt extends Algebra {
```

Comment:
Why is this called `ClipIt`? That doesn't follow any of the existing naming convention. Just `Clip` will do.